24HW/src/ui/step-definitions/products/details-product.steps.ts

```diff
@@ -0,0 +1,21 @@
+import { Then} from "@wdio/cucumber-framework";
+import _ from "lodash";
+import productsPageService from "../../services/Products/productsPage.service";
+import DetailsProductModal from "../../pages/Products/details.modal";
+
+Then(/^I check "([^"]*)" of the product on "Details Modal" page$/, async function (value: string) {
+    const createdProduct = this.createdProduct;
+    const actualProductData = await DetailsProductModal.getActualProductData();
+    expect(createdProduct[value]).toBe(actualProductData[value]);
+  });
+  
+Then(/^I close modal on "Details Modal" page$/, async function () {
```

Comment:
```suggestion
Then(/^I close "Details" modal on "Products" page$/, async function () {
```
Можно в дальнейшем сделать кастомньій степ, чтоб закрьівать любое модальное окно внезависимости от страницьі